etna/search/tests/test_views.py

```diff
@@ -606,348 +595,587 @@ def test_current_bucket(self):
+                "customDimension8": "All results: none",
+                "customDimension9": "test search term",
+                "customDimension10": "",
+                "customDimension11": "",
+                "customDimension12": "",
+                "customDimension13": "",
+                "customDimension14": "",
+                "customDimension15": "",
+                "customDimension16": "",
+                "customDimension17": "",
+                "customMetric1": 11,
+                "customMetric2": 0,
+            },
         )
-        with open(path, "r") as f:
-            responses.add(
-                responses.GET,
-                "https://kong.test/data/searchAll",
-                json=json_module.loads(f.read()),
-            )
-
-        response = self.client.get("/search/featured/?q=test+search+term")
-
-        self.assertTemplateUsed(response, "search/featured_search.html")
-
-        html_decoded_response = response.content.decode("utf8")
-        desired_datalayer_script_tag = """<script id="gtmDatalayer" type="application/json">{"contentGroup1": "Search", "customDimension1": "offsite", "customDimension2": "", "customDimension3": "FeaturedSearchView", "customDimension4": "", "customDimension5": "", "customDimension6": "", "customDimension7": "", "customDimension8": "All results: none", "customDimension9": "test search term", "customDimension10": "", "customDimension11": "", "customDimension12": "", "customDimension13": "", "customDimension14": "", "customDimension15": "", "customDimension16": "", "customDimension17": "", "customMetric1": 11, "customMetric2": 0}</script>"""
-        self.assertIn(desired_datalayer_script_tag, html_decoded_response)
 
     @responses.activate
     def test_datalayer_catalogue_search_tna(self):
-        path = (
-            f"{settings.BASE_DIR}/etna/search/tests/fixtures/catalogue_search_tna.json"
+        self.assertDataLayerEquals(
+            path=reverse("search-catalogue"),
+            query_data={},
+            api_resonse_path=f"{settings.BASE_DIR}/etna/search/tests/fixtures/catalogue_search_tna.json",
+            expected={
+                "contentGroup1": "Search",
+                "customDimension1": "offsite",
+                "customDimension2": "",
+                "customDimension3": "CatalogueSearchView",
+                "customDimension4": "",
+                "customDimension5": "",
+                "customDimension6": "",
+                "customDimension7": "",
+                "customDimension8": "Catalogue results: tna",
+                "customDimension9": "*",
+                "customDimension10": "",
+                "customDimension11": "",
+                "customDimension12": "",
+                "customDimension13": "",
+                "customDimension14": "",
+                "customDimension15": "",
+                "customDimension16": "",
+                "customDimension17": "",
+                "customMetric1": 10001,
```

Comment:
```suggestion
                "customMetric1": 10000,
```
to resolve and align with issue of consistency for `customMetric1` in this PR.

Reply:
Resolved in [a0ccd87](https://github.com/nationalarchives/ds-wagtail/pull/422/commits/a0ccd87f3aca7f9f87788eaccc56a67f9f38fdb9)